wolf-data/src/Wolf/Data.hs

```diff
@@ -6,6 +6,10 @@ module Wolf.Data
     , initWolf
     , withInitCheck
     , withInitCheck_
+    -- * Prompt
+    , promptYesNo
```

Comment:
Wolf doesn't need to export these prompts. This is not wolf data functionality.
There is a reason this is a seperate import, and this is definitely a seperate refactor and not to be done here.